src/SwaggerUI/Silex/Provider/SwaggerUIServiceProvider.php

```diff
@@ -21,39 +20,38 @@ class SwaggerUIServiceProvider implements ServiceProviderInterface
      */
     public function boot(Application $app)
     {
-        $app->get($app['swaggerui.path'], function(Request $request) use ($app) {
-            return str_replace(
-                ['{{swaggerui-root}}', '{{swagger-docs}}'],
-                [$request->getBasePath() . $app['swaggerui.path'], $app['swaggerui.apiDocPath']],
-                file_get_contents(__DIR__ . '/../../../../public/index.html')
-            );
-        });
+        $self = &$this;
 
-        $app->get($app['swaggerui.path'] . '/{resource}', function($resource) use ($app) {
-            $file = __DIR__ . '/../../../../public/' . $resource;
-            if (is_file($file)) {
-                return file_get_contents($file);
-            }
+        $app->get($app['swaggerui.path'], function() use ($app) {
 
-            return '';
+
+            return str_replace(array('{{swaggerui-root}}', '{{swagger-docs}}'), array($app['request']->getBaseUrl().$app['swaggerui.path'],$app['request']->getBaseUrl(). $app['swaggerui.apiDocPath']), file_get_contents(__DIR__ . '/../../../../public/index.html'));
```

Comment:
Please use space before and after `.` when concatenating strings to be consistent with existing code.